worlds/ufo50/night-manor/items.py

```diff
@@ -0,0 +1,135 @@
+    "Flashlight": ItemInfo(17, IC.progression),
+    "Duct Tape": ItemInfo(18, IC.progression),
+    "Journal Entry 8": ItemInfo(19, IC.filler, "journal_entries"),
+    "Gas Can": ItemInfo(20, IC.progression),
+    "Crowbar": ItemInfo(21, IC.progression),
+    "Ornamental Egg": ItemInfo(22, IC.progression),
+    "Red Gemstone": ItemInfo(23, IC.progression, "gems"),
+    "Journal Entry 9": ItemInfo(24, IC.filler, "journal_entries"),
+    "Pool Cue": ItemInfo(25, IC.progression),
+    "Journal Entry 10": ItemInfo(26, IC.filler, "journal_entries"),
+    "Sheet Music": ItemInfo(27, IC.progression),
+    "Copper Key": ItemInfo(28, IC.progression, "keys"),
+    "Screwdriver": ItemInfo(29, IC.progression), 
+    "Journal Entry 11": ItemInfo(30, IC.filler, "journal_entries"),
+    "Wrench": ItemInfo(31, IC.progression),
+    "Hedge Shears": ItemInfo(32, IC.progression),
+    "Shovel": ItemInfo(33, IC.progression),
+    "Bronze Key": ItemInfo(34, IC.progression, "keys"),
+    "Gold Key": ItemInfo(35, IC.progression, "keys"),
+    "Motor": ItemInfo(36, IC.progression),
+    "Steel Key": ItemInfo(37, IC.progression, "keys"),
+    "Hacksaw": ItemInfo(38, IC.progression),
+    "Silver Key": ItemInfo(39, IC.progression, "keys"),
+    "Ring": ItemInfo(40, IC.progression),
+    "Brass Key": ItemInfo(41, IC.progression, "keys"),
+    "Gear": ItemInfo(42, IC.progression),
+    "Green Gemstone": ItemInfo(43, IC.progression, "gems"),
+    "Journal Entry 12": ItemInfo(44, IC.filler, "journal_entries"),
+    "Journal Entry 15": ItemInfo(45, IC.filler, "journal_entries"),
+    "Journal Entry 6": ItemInfo(46, IC.filler, "journal_entries"),
+    "Magnifying Glass": ItemInfo(47, IC.progression),
+    "Tea Tree Oil": ItemInfo(48, IC.progression),
+    "Hydrogen Peroxide": ItemInfo(49, IC.progression),
+    "Safe Combination": ItemInfo(50, IC.progression),
+    "Journal Entry 16": ItemInfo(51, IC.filler, "journal_entries"),
+    "Cigar Butt": ItemInfo(52, IC.progression),
+    "Yellow Gemstone": ItemInfo(53, IC.progression, "gems"),
+    "Journal Entry 13": ItemInfo(54, IC.filler, "journal_entries"),
+    "Computer Password": ItemInfo(55, IC.progression),
+    "Journal Entry 3": ItemInfo(56, IC.filler, "journal_entries"),
+    "Piano Wire": ItemInfo(57, IC.progression),
+    "Crossbow": ItemInfo(58, IC.progression),
+    "Doll": ItemInfo(59, IC.progression),
+    "Aluminum Key": ItemInfo(60, IC.progression, "keys"),
+    "Journal Entry 14": ItemInfo(61, IC.filler, "journal_entries"),
+    "Fungicide Recipe": ItemInfo(62, IC.progression),
+    "Fungicide": ItemInfo(63, IC.progression),
+    "White Gemstone": ItemInfo(64, IC.progression, "gems"),
+    "Glasses": ItemInfo(65, IC.progression),
+    "Maze Directions": ItemInfo(66, IC.progression),
+    "Crossbow Bolt": ItemInfo(67, IC.progression),
+    "Iron Key": ItemInfo(68, IC.progression, "keys"),
+    "Journal Entry 17": ItemInfo(69, IC.filler, "journal_entries")
+
+}
+
+night_manor_item_groups; Dict[Dict[str]] = {
```

Comment:
```suggestion
night_manor_item_groups: Dict[str, Set[str]] = {
```
And you need to import Set from typing

Reply:
Thanks, will commit the suggestion and import Set when I'm home 👍